jobs/src/main/scala/dcos/metronome/model/CronSpec.scala

```diff
@@ -20,12 +20,12 @@ class CronSpec(val cron: Cron) {
 
   def nextExecution(from: ZonedDateTime): ZonedDateTime = {
     val fromDateTime: ThreeTenZonedDateTime = javaTimeToThreetenTime(from)
-    threetenToJavaTime(executionTime.nextExecution(fromDateTime).get())
+    executionTime.nextExecution(threetenToJavaTime(fromDateTime)).get()
```

Comment:
Can someone have a special look at this change as well?

It didn't compile for me what we have in master.

Reply:
@kensipe

Reply:
master didn't compile?   that seems strange.. it compiles locally and on jenkins.    just tested again... you sure?

Reply:
if you switched while using IDEA and didn't refresh the sbt... this would have happened... could you check again?

Reply:
regarding the change... cron-utils 7.0.0 must have move away from 310 and strictly to Java dates... I will take a look after this PR.  We may not need the conversations anymore.   The frustration is that they have had 9 releases without release notes with 6.0.3 being the last documented release 👎 .

Reply:
Yes I believe I missed an sbt reload.

Reply:
Yes I believe I missed an sbt reload.